conbench/app/_plots.py

```diff
@@ -186,21 +192,25 @@ def _source(
 ):
     # Note(JP): important magic: if not specified otherwise, this extracts
     # the mean-over-time.
-    key = "distribution_mean" if distribution_mean else "mean"
-
     unit_fmt = formatter_for_unit(unit)
 
     # TODO: These commit message prefixes end up in the on-hover tooltip.
     # Change this to use short commit hashes. The long commit message prefix
     # does not unambiguously specify the commit. Ideally link to the commit, in
     # the tooltip?
-    commit_messages = [d["message"] for d in data]
+    commit_messages = [s.commit_msg for s in samples]
```

Comment:
"message" became "commit_msg". Tiny improvements like this make code more readable.